opendebates/moderator_views.py

```diff
@@ -46,15 +47,12 @@ def mark_duplicate(request):
         votes_to_merge.update(original_merged_submission=to_remove, submission=duplicate_of)
 
     if request.POST.get("send_email") == "yes":
-        # FIXME: `emails` was not defined, so I have commented it out for now. --vkurup
         if request.POST.get("handling") == "merge":
-            pass
-            # emails.idea_is_merged(to_remove)
+            send_email("your_idea_is_merged", {"idea": to_remove})
```

Comment:
It seems like all of these need to pass in the idea submitter's email address so send_email will know where to send the mail to.